internal/provider/datasource_cloud_conn.go

```diff
@@ -353,6 +386,21 @@ func flattenCloudProvider(provider *packetfabric.AwsCloudProvider) []interface{}
 	return flattens
 }
 
+func flattenBgpStateList(bgp_state_list *[]packetfabric.BgpStateObj) []interface{} {
```

Comment:
The convention in Go is to use `mixedCaps` for everything, not `snake_case`. Please correct the name.